docker/alpine-latest-amd64-openssl-3.0.x/run.sh

```diff
@@ -30,3 +28,5 @@ docker run -p 3000:3000 \
   "${PRISMA_DOCKER_IMAGE_NAME}" &
 
 sleep 15
+
+yarn install
```

Comment:
Any specific reason why this moved?

Reply:
`yarn install` is run on the host machine (so, not Docker) and will always install the engines of Prisma that match the Github Actions' runner. If `yarn install` is run before `docker buildx ...`, Docker takes longer to build and run its image because the local context is heavier (due to `node_modules`). This local installation of Prisma is only needed after `./run.sh` has run anyway, so we're not changing any behavior, we're just avoiding slowing down the pipeline (which is also particularly handy locally)

Reply:
Makes a lot of sense! thanks!

Reply:
For things unrelated to this PR. From what I have read skimming around, for reporting the version of Prisma installed and for running some js scripts after each test

Reply:
Might not really make sense in Docker context. Probably there is a denylist to run those...

But indeed not necesary to fix right now.